en/docs/ipaas/micro-integrator/develop-integrations-with-integration-studio.md

```diff
@@ -45,7 +45,7 @@ Adding third-party JARs to the Micro Integrator can provide the following benefi
 - Providing optimized implementations of common functions, improving the performance of Micro Integrator.
 - Developers can avoid having to re-implement commonly used functions, reducing development time and increasing efficiency.
 
-To add third-party libraries to the Micro Integrator, you can add a new directory named `libs` to the root of the Micro Integrator project. When you deploy the component, all the JARs in the `libs` directory in the project path will be included in the Micro Integrator runtime.
+To add third-party libraries to the Micro Integrator, create a new directory named `libs` within the root of the Micro Integrator project. It is important to note that this directory should be placed specifically at the project root, rather than the root of the GitHub repository. When you deploy the component, all the JARs in the `libs` directory in the project path will be included in the Micro Integrator runtime.
```

Comment:
This sentence tells you that you should add the directory at the project root only. If we want to tell that in the case where the GitHub repo root and project root are not the same add it in project root, we can opt for one of the two below: 
1) To incorporate third-party libraries into your Micro Integrator project, you need to create a new directory called libs at the root of your project. If your project root and GitHub repository root are not the same, you should place the libs directory in the project root.
Once you have placed the necessary JAR files in the libs directory, the Micro Integrator runtime will automatically include them when you deploy the component. This ensures that any dependencies required by your integration flow are available at runtime.
2) To add third-party libraries to Micro Integrator, create a libs directory at the project root. If the GitHub root differs, place the folder at the project root. The Micro Integrator runtime will include all JARs in the libs directory upon component deployment.

Reply:
Let's go with option 1. Fixed in - 64478c44d2a71215636fe924cae4d374f935528a